tests/test_utils.py

```diff
@@ -660,6 +660,30 @@ async def test_extract_text(self, mock_responses):
 
             assert response == "I've been extracted!"
 
+    @pytest.mark.asyncio
+    async def test_extract_text_with_file_pointer(self, mock_responses):
+        filepath = "/tmp/notreal.txt"
+        url = "http://localhost:8090/extract_text/?local_file_path=/tmp/notreal.txt"
+        payload = {"extracted_text": "I've been extracted!"}
```

Comment:
```suggestion
        payload = {"extracted_text": "I've been extracted from a local file!"}
```

Reply:
The fact that this test was passing is concerning, I'll investigate first.